batchflow/models/torch/base.py

```diff
@@ -122,18 +120,19 @@ class TorchModel:
         Examples:
 
         - ``{'optimizer': 'Adam'}``
-        - ``{'optimizer': ('SparseAdam', {'lr': 0.01})}``
-        - ``{'optimizer': {'name': 'Adagrad', 'initial_accumulator_value': 0.01}``
-        - ``{'optimizer': {'name': MyCustomOptimizer, momentum=0.95}}``
+        - ``{'optimizer': {'name': 'SparseAdam', 'lr': 0.01}}``
+        - ``{'optimizer': {'name': 'Adagrad', 'initial_accumulator_value': 0.01}}``
+        - ``{'optimizer': {'name': MyCustomOptimizer, 'momentum': 0.95}}``
 
-    decay : str, tuple, dict
+    decay : dict, list
         Learning rate decay algorithm, might be defined in multiple formats.
-        All decays require to have ``n_iters`` as a key in a configuration
+        All decays require to have 'frequency' as a key in a configuration
```

Comment:
It has nothing to do with epochs: it just defines how often you apply `decay step`